sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -489,4 +491,12 @@ public static void setFeedOptionsMaxItemCount(FeedOptions feedOptions, Integer m
     public static <T> CosmosContinuablePagedFlux<T> createCosmosContinuablePagedFlux(Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> pagedFluxOptionsFluxFunction) {
         return new CosmosContinuablePagedFlux<>(pagedFluxOptionsFluxFunction);
     }
+
+    public static Mono<FeedResponse<Document>> readMany(CosmosAsyncContainer container, 
+                                                     List<Pair<String, PartitionKey>> itemKeyList){
```

Comment:
>String, PartitionKey [](start = 63, length = 20)

Parameter order consistency w.r.t. other API's

Reply:
This API is kind of different from the other we have. Still we are sort of consistent